scripts/camViewer

```diff
@@ -127,6 +127,18 @@ iocfpv() {
      grep_pv $1 | sed -n 's/\/reg\/d\/iocData\/\(\S*\)\/iocInfo/\1/p'
 }
 
+iocinfo() {
+     for ioc in `iocfpv "$CAMPVFULL:IMAGE1"`; do
+
+        INFO=$(grep_ioc "$ioc" all | grep "id:'$ioc'")
+        if [ -z "$INFO" ]; then
+            :            
+        else
+            iochutch=$(echo $INFO | cut -d'/' -f7)
+        fi
```

Comment:
More compact way to handle these sorts of cases. `-n` is effectively the opposite of `-z`
```suggestion
        if [ -n "$INFO" ]; then
            iochutch=$(echo $INFO | cut -d'/' -f7)
        fi
```